src/HermsManagement.py

```diff
@@ -0,0 +1,76 @@
+from discord.ext import commands
+import logging
+import discord

```

Comment:
To keep dependencies organized as things grow, I usually do:

1. Built-in packages, in alphabetical order
2. Empty line
3. Non built-in packages, in alphabetical order

Reply:
(do this for all the files you're adding)